src/pyodide/internal/setupPackages.js

```diff
@@ -174,7 +174,9 @@ function addPackageToLoad(lockfile, name, toLoad) {
   }
   const pkgInfo = lockfile.packages[normalizedName];
   if (!pkgInfo) {
-    throw new Error(`No known package with name '${name}'`);
+    throw new Error("It appears that a package (" + normalizedName + ") you requested is not available yet on workerd. \n" +
```

Comment:
Maybe use `name` instead of `normalizedName` in the message:
```suggestion
    throw new Error(`It appears that a package ("${name}") you requested is not available yet in workerd. \n` +
```